Source/JavaScriptCore/llint/LLIntSlowPaths.h

```diff
@@ -137,6 +137,7 @@ LLINT_SLOW_PATH_HIDDEN_DECL(slow_path_handle_traps);
 LLINT_SLOW_PATH_HIDDEN_DECL(slow_path_debug);
 LLINT_SLOW_PATH_HIDDEN_DECL(slow_path_handle_exception);
 LLINT_SLOW_PATH_HIDDEN_DECL(slow_path_get_from_scope);
+LLINT_SLOW_PATH_HIDDEN_DECL(slow_path_rgs_get_from_scope);
```

Comment:
nit: same thing about writing out the full name

Reply:
Updated